src/Stepper/Stepper.scss

```diff
@@ -13,6 +13,7 @@
   justify-content: center;
   background: $white;
   padding: .75rem 1rem;
+  min-height: 4rem;
```

Comment:
@AhtishamShahid Does this new height match the Figma design?

Reply:
There is a difference of few decimal points in pixels so I guess it is negligible .